facade/src/main/java/org/jboss/pnc/facade/deliverables/DeliverableAnalyzerManagerImpl.java

```diff
@@ -259,6 +259,7 @@ private org.jboss.pnc.model.Artifact findOrCreateArtifact(
 
         // create
         artifact.setTargetRepository(targetRepo);
+        createGenericPurl(targetRepo.getRepositoryPath(), artifact.getFilename().toString(), artifact.getSha256());
```

Comment:
Do you just throw away the generated purl?

Reply:
From my understanding of the code, at this point there is no generated PURL for the not found artifacts yet, so I am not throwing anything away?

Reply:
The method returns String and has no sideffect (except for the posible exception) and the return value is not used. So this line basically does nothing?

Reply:
doh of course